Makefile

```diff
@@ -242,7 +242,12 @@ deploy-images:
 .PHONY: golint
 golint:
 	@echo "--> Running golint"
-	@echo "Consider these linter recommendations:"
-	@golint $(PACKAGES)
+	@for file in ${PACKAGES}; do \
+		golint $${file}; \
```

Comment:
why this change?

Reply:
I wanted to loop it so that the "consider the recommendations" only shows up when there is a file that has a recommendation.

Reply:
I will just try to achieve the same result with the $(PACKAGES) again and let you know if there are blockers ... I don't remember exactly why I was doing this anymore too, sorry

Reply:
oh now i remember, basically with it this way I couldn't get it to print "Consider these linter recommendations:" before the recommendations. so i did it this way thats all.

Reply:
@vaniisgh the problem with the earlier approach was, as soon as it gets linting error in 1 file, it exits. So developer has to keep on firing the `make golint` command until there is no error. It is better to print all the errors at once.

can we use `@golint -set_exit_status ${PACKAGES}` it will fail if there is linting error, otherwise it will pass.

Reply:
oh now i remember, basically with it this way I couldn't get it to print "Consider these linter recommendations:" before the recommendations. so i did it this way thats all .

this sounds way better!